Chapter1/3-ElementBufferObjects/Window.cs

```diff
@@ -89,6 +86,7 @@ protected override void OnRenderFrame(FrameEventArgs e)
 
             _shader.Use();
 
+            // This call will populate the current ElementArrayObject with the one stored in the VAO.
```

Comment:
hmmm this sounds confusing here, since we just tell opengl - hey gl, use this vao... which is still bound

Reply:
Rewrote to be more hopefully more clear.